api/db-cdr/generate-cdr/make-bq-data-dictionary.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# This generates the big query data dictionary tables by importing data from
+# csv at gs://all-of-us-workbench-private-cloudsql/$BQ_DATASET/data_dictionary.
+
+set -ex
+
+export BQ_PROJECT=$1  # project
+export BQ_DATASET=$2  # dataset
+export DRY_RUN=$3     # dry run
+
+archiveList=$(gsutil ls gs://all-of-us-workbench-private-cloudsql/$BQ_DATASET/data_dictionary/archive/*.csv) > /dev/null || true;
+dataDictionaryList=$(gsutil ls gs://all-of-us-workbench-private-cloudsql/$BQ_DATASET/data_dictionary/*.csv) > /dev/null || true;
+
+if [ "$DRY_RUN" == true ]
+then
+  echo "Check if data dictionary file exists"
+  if [ -z "$dataDictionaryList" ] && [ -z "$archiveList" ]
+  then
+    echo "Error: Data Dictionary file is missing!"
+    exit 1;
+  fi
+  exit 0
+fi
+
+# Throw error if there is no data_dictionary file for CDR release
+echo "Check if data dictionary file exists"
+if [ -z "$dataDictionaryList" ] && [ -z "$archiveList" ]
+then
+  echo "Error: Data Dictionary file is missing!"
+  exit 1;
+fi
+
+tableCreated=false;
+
+for filename in $dataDictionaryList; do
+  if (! $tableCreated)
+  then
+    echo "CREATE TABLE - ds_data_dictionary"
+    bq --quiet --project_id=$BQ_PROJECT query --nouse_legacy_sql \
+    "CREATE OR REPLACE TABLE \`$BQ_PROJECT.$BQ_DATASET.ds_data_dictionary\`
```

Comment:
Instead of creating the table inline i would move it to the bq-schemas directory as a json file that way we can reuse this in shell scripts down the line instead of duplicating: https://github.com/all-of-us/workbench/blob/bcddab1b0ce2f0aa105aabd2668e39f030b3b793/api/db-cdr/generate-cdr/bq-schemas#L90-L90

This way you can create tables like so:
`schema_path=generate-cdr/bq-schemas
create_tables=(ds_data_dicitionary)

for t in "${create_tables[@]}"
do
    bq --project_id=$BQ_PROJECT rm -f $BQ_DATASET.$t
    bq --quiet --project_id=$BQ_PROJECT mk --schema=$schema_path/$t.json $BQ_DATASET.$t
done`